xlsx/tests/csv.rs

```diff
@@ -0,0 +1,40 @@
+use std::io::Read;
+use std::{env, fs, io};
+use uuid::Uuid;
+
+use ironcalc::compare::{test_file, test_load_and_saving};
+use ironcalc::export::save_to_xlsx;
+use ironcalc::import::{load_from_icalc, load_from_xlsx, load_from_xlsx_bytes};
+use ironcalc_base::types::{HorizontalAlignment, VerticalAlignment};
+use ironcalc_base::Model;
+
+use std::process::{Command, ExitStatus};
+
+#[test]
+fn test_simple_csv() {
```

Comment:
There are a huge number of edge cases in CSV files that you'll want to test. Escaped (or unescaped) commas inside strings, function calls, misaligned column count, etc.